landing-page/public/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://json.schemastore.org/web-manifest-combined.json",
+  "name": "helpwave",
+  "short_name": "helpave",
+  "lang": "en",
+  "description": "Next generation healthcare software approach!",
+  "orientation": "portrait",
+  "categories": ["software", "medical", "health", "healthcare", "company"],
+  "scope": "https://helpwave.de/",
+  "start_url": "https://helpwave.de",
+  "background_color": "#281c20",
+  "theme_color": "#281c20",
```

Comment:
uncertain about these values, why was the background color chosen as the theme color as well?

Reply:
The `theme_color` is the one used for the browser frame and application switcher on mobile.
The `background_color` is an experimental attribute which defines the background color before the stylesheet has loaded. [See here](https://developer.mozilla.org/en-US/docs/Web/Manifest/background_color) 
So both should use the same color (in this case the dark-red-brown tone `#281c20`).

Reply:
works for me :+1: